app/src/main/res/values/strings.xml

```diff
@@ -1028,6 +1028,8 @@
     <!-- Content description (not visible, for screen readers etc.):
         "Share" button. Opens the share menu when pressed. -->
     <string name="share_button_content_description">Share</string>
+    <!-- Share to PDF button -->
```

Comment:
nit: add some more detail description for translators to know where to find the feature.
```suggestion
    <!-- Text for the Save to PDF feature in the share sheet -->
```